documents/MQTT5_Userguide.md

```diff
@@ -16,12 +16,15 @@
         - [MQTT over Websockets with Sigv4 authentication](#mqtt-over-websockets-with-sigv4-authentication)
         - [Direct MQTT with Custom Authentication](#direct-mqtt-with-custom-authentication)
         - [MQTT over Websockets with Cognito](#mqtt-over-websockets-with-cognito)
-    + [Adding an HTTP Proxy](#adding-an-http-proxy)
-    + [Client Operations](#client-operations)
-        - [Subscribe](#subscribe)
-        - [Unsubscribe](#unsubscribe)
-        - [Publish](#publish)
-* [MQTT5 Best Practices](#mqtt5-best-practices)
+        - [Direct MQTT with Windows Certificate Store Method](#direct-mqtt-with-windows-certificate-store-method)
+        - [Direct MQTT with PKCS11 Method](#direct-mqtt-with-pkcs11-method)
+        - [direct mqtt with pkcs12 method](#direct-mqtt-with-pkcs12-method)
+    + [adding an http proxy](#adding-an-http-proxy)
+    + [client operations](#client-operations)
+        - [subscribe](#subscribe)
+        - [unsubscribe](#unsubscribe)
+        - [publish](#publish)
+* [mqtt5 best practices](#mqtt5-best-practices)
```

Comment:
The titles were capitalized. It might be better to keep it consistant. 
"adding" -> "Adding"
"client" -> "Client"
"subscribe" -> "Subscribe"
"unsubscribe" -> "Unsubscribe"
"publish" -> "Publish"
"mqtt5" -> "Mqtt5"